### ISSUE TITLE
Get 10 - 15 prompts when you edit project inside VS to add an import to a non-existent targets file

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn-project-system#1407]
1. File -> New Console App (.NET Core)
2. Right-click -> Edit
3. Add 
``` XML
<Import Project="IDoNotExist.targets" />
```
4. Save

Expected: An error in the error about missing import
Actual:

10 -> 15 prompts of this dialog:

```
---------------------------
Microsoft Visual Studio
---------------------------
The project 'ConsoleApp2' has encountered an error.  You may need reload the project after fixing the problem. 

The imported project "c:\users\davkean\documents\visual studio 2017\Projects\SharedProject10\ConsoleApp2\Foo.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.  c:\users\davkean\documents\visual studio 2017\Projects\SharedProject10\ConsoleApp2\ConsoleApp2.csproj
---------------------------
OK   
---------------------------
```

**Comment by davkean** [comment]
Dup'ing against: https://github.com/dotnet/roslyn-project-system/issues/1570
